pkg/podvolume/restorer.go

```diff
@@ -129,6 +145,8 @@ func (r *restorer) RestorePodVolumes(data RestoreData) []error {
 	r.results[resultsKey(data.Pod.Namespace, data.Pod.Name)] = resultsChan
 	r.resultsLock.Unlock()
 
+	r.nodeAgentCheck = make(chan struct{})
```

Comment:
how about making it an error channel so we have more information of the node name in the error message?